fixture.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+
+var loudRejection = require('./');
+var Promise = require('bluebird');
```

Comment:
`Promise` is, but `unhandledRejection` events are not emitted by native `0.12` promises